### DISCUSSION TITLE
Allow searching for GitHub URLs

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#125564] [Mobile]
In the GitHub mobile app, please go to the homepage, by the button in the lower left corner. Now at the top of the screen you'll see a search bar. 

Now enter for example, 

```
https://github.com/osmlab/osm-community-index/issues/715
```

to search for it. 

Do you ever arrive at [it](https://github.com/osmlab/osm-community-index/issues/715)? No.

**Comment by taki-on** [comment]
@jidanni We do not support searching by urls, but it is an interesting idea. Tapping on the url outside the app will open the app and navigate to the item (by deep linking), though. Have you tried it?

**Reply by jidanni** [reply]
Thanks, but as navigating from outside fails for some people
https://github.com/orgs/community/discussions/124987
it would be good to allow searching from within the app as a backup method!
